components/layout/display_list_builder.rs

```diff
@@ -1440,7 +1440,7 @@ impl FragmentDisplayListBuilding for Fragment {
     fn adjust_clipping_region_for_children(&self,
                                            current_clip: &mut ClippingRegion,
                                            stacking_relative_border_box: &Rect<Au>,
-                                           is_absolutely_positioned: bool) {
+                                           _is_absolutely_positioned: bool) {
```

Comment:
If we're not using this argument, it's either a bug or we should remove it :)

cc @mbrubeck